packages/gatsby/src/utils/gatsby-webpack-stats-extractor.ts

```diff
@@ -25,11 +26,33 @@ export class GatsbyWebpackStatsExtractor {
                 f.slice(0, chunkGroup.name.length) === chunkGroup.name
             )
             .map(filename => `/${filename}`)
+
+          for (const [rel, childChunkGroups] of Object.entries(
+            chunkGroup.getChildrenByOrders(
+              stats.compilation.moduleGraph,
+              stats.compilation.chunkGraph
+            )
+          )) {
+            if (!(chunkGroup.name in childAssets)) {
+              childAssets[chunkGroup.name] = {}
+            }
+
+            const childFiles = []
+            for (const childChunkGroup of childChunkGroups) {
+              for (const chunk of childChunkGroup.chunks) {
+                childFiles.push(...chunk.files)
+              }
+            }
+
+            childAssets[chunkGroup.name][rel] = childFiles
+          }
         }
       }
+
       const webpackStats = {
         ...stats.toJson({ all: false, chunkGroups: true }),
```

Comment:
`gatsby` itself no longer need those, so we maybe could remove it, I'm just not sure if some plugins might rely on this, so it IS safer to keep it :/